<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-12-03T07:28:30 IST" name="Default suite" finished-at="2021-12-03T07:28:35 IST" duration-ms="5229">
    <groups>
    </groups>
    <test started-at="2021-12-03T07:28:30 IST" name="Default test" finished-at="2021-12-03T07:28:35 IST" duration-ms="5229">
      <class name="Dishnetworks.RestAssured.SimpleGetTest">
        <test-method signature="GetInvalidResponse()[pri:0, instance:Dishnetworks.RestAssured.SimpleGetTest@358ee631]" started-at="2021-12-03T07:28:30 IST" name="GetInvalidResponse" finished-at="2021-12-03T07:28:32 IST" duration-ms="2268" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Correct status code returned expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Correct status code returned expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at Dishnetworks.RestAssured.SimpleGetTest.GetInvalidResponse(SimpleGetTest.java:59)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetInvalidResponse -->
        <test-method signature="GetStatusLine()[pri:0, instance:Dishnetworks.RestAssured.SimpleGetTest@358ee631]" started-at="2021-12-03T07:28:32 IST" name="GetStatusLine" finished-at="2021-12-03T07:28:33 IST" duration-ms="934" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetStatusLine -->
        <test-method signature="GetresponseBody()[pri:0, instance:Dishnetworks.RestAssured.SimpleGetTest@358ee631]" started-at="2021-12-03T07:28:33 IST" name="GetresponseBody" finished-at="2021-12-03T07:28:34 IST" duration-ms="697" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetresponseBody -->
        <test-method signature="Getstatuscode()[pri:0, instance:Dishnetworks.RestAssured.SimpleGetTest@358ee631]" started-at="2021-12-03T07:28:34 IST" name="Getstatuscode" finished-at="2021-12-03T07:28:35 IST" duration-ms="607" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Getstatuscode -->
        <test-method signature="ValidateResponseBody()[pri:0, instance:Dishnetworks.RestAssured.SimpleGetTest@358ee631]" started-at="2021-12-03T07:28:35 IST" name="ValidateResponseBody" finished-at="2021-12-03T07:28:35 IST" duration-ms="674" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidateResponseBody -->
      </class> <!-- Dishnetworks.RestAssured.SimpleGetTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
